Change the default SSL directory, as buildroot ca-cert package is
installed at /etc/ssl. That way, we don't have to use the SSL_CERT_DIR
environment at runtime.

Signed-off-by: Damien Riegel <damien.riegel@savoirfairelinux.com>

diff -Nura qt5webengine-5.8.0.orig/src/core/qtwebengine_extras.gypi qt5webengine-5.8.0/src/core/qtwebengine_extras.gypi
--- qt5webengine-5.8.0.orig/src/core/qtwebengine_extras.gypi	2017-01-03 05:35:11.000000000 -0500
+++ qt5webengine-5.8.0/src/core/qtwebengine_extras.gypi	2017-03-03 16:24:34.947416678 -0500
@@ -103,7 +103,7 @@
             'GL_GLEXT_PROTOTYPES',
             'EGL_EGLEXT_PROTOTYPES',
             # At runtime the env variable SSL_CERT_DIR can be used to override this
-            'OPENSSLDIR="/usr/lib/ssl"',
+            'OPENSSLDIR="/etc/ssl"',
             'OPENSSL_LOAD_CONF',
             'EGL_API_FB=1',
             'LINUX=1',
